src/EssentialSigner/index.ts

```diff
@@ -40,42 +55,70 @@ export class EssentialSigner extends Signer {
     });
   }
 
-  // Populates "from" if unspecified, and calls with the transaction
-  async call(
-    transaction: Deferrable<TransactionRequest>,
-    blockTag?: BlockTag,
-  ): Promise<string> {
-    this._checkProvider('call');
-    const tx = await resolveProperties(this.checkTransaction(transaction));
-    return await this.provider.call(tx, blockTag);
-  }
-
   // Populates all fields in a transaction, signs it and sends it to the network
   async sendTransaction(
     transaction: Deferrable<TransactionRequest>,
-  ): Promise<TransactionResponse> {
-    this._checkProvider('sendTransaction');
-    const tx = await this.populateTransaction(transaction);
-    const signedTx = await this.signTransaction(tx);
-    return await this.provider.sendTransaction(signedTx);
+  ): Promise<TransactionResponse | any> {
+    const result = await signMetaTxRequest(
+      this.provider || this.privateKey,
+      // for us, this chain ID is the chain where our Forwarding and implementation contract live.
+      // our payload is network-agnostic, so it won't necessarily be the same chain as the provider.
+      parseInt(process.env.CHAIN_ID!, 10),
```

Comment:
another but that feels like a global config thing?

we can't depend on the provider's network, we want things to be network-agnostic.

we do however need this piece of data as the chain ID is verified as part of signatures to ensure we are only accepting meta-txs on the network they are intended for